examples/jdbc-execute-operation/jdbc_execute_operation.md

```diff
@@ -4,7 +4,7 @@ This BBE demonstrates how to use the JDBC client with DDL and DML operations. No
 
 >**Note:** This sample is based on an H2 database and the H2 database driver JAR needs to be added to the `Ballerina.toml` file. This sample is written using H2 2.0.6 and it is recommended to use an H2 JAR file of a version higher than 2.0.2.
 
-For a sample configuration and more information on the underlying module, see the [`jdbc` module](https://lib.ballerina.io/ballerinax/java.jdbc/latest/) .
+For a sample configuration and more information on the underlying module, see the [`jdbc` module](https://lib.ballerina.io/ballerinax/java.jdbc/latest/).
```

Comment:
```suggestion
For a sample configuration and more information on the underlying module, see the [`jdbc` module](https://lib.ballerina.io/ballerinax/java.jdbc/latest/).
```
We are changing this wording. Please check https://github.com/ballerina-platform/ballerina-distribution/pull/3563/files.

Reply:
@praneesha Acck will send separate PR